cluster/log-dump/log-dump.sh

```diff
@@ -419,6 +419,10 @@ function dump_nodes() {
   all_selected_nodes+=( "${windows_node_names[@]}" )
 
   proc=${max_dump_processes}
+  start="$(date +%s)"
+  # This is just a rough estimation of the timeout length which may be
+  # visibly exceeded due to waiting for all the child processes below.
+  log_dump_ssh_timeout="${LOG_DUMP_SSH_TIMEOUT:=}"
```

Comment:
NIT: Consider adding unit as a suffix of the env, e.g. `LOG_DUMP_SSH_TIMEOUT_SECONDS`

Reply:
Done.